app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -139,20 +128,19 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
             toggleNav.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
                 @Override
                 public boolean onNavigationItemSelected(@NonNull MenuItem item) {
-                    // reset everytime user switches between trending and recommended
-                    adapter.clear();
-
                     switch (item.getItemId()) {
                         case R.id.action_trending:
                             // get trending shows
-                            TraktApplication.fetchTrendingShows(showsObj, new ShowCallback());
+                            adapter.clear();
+                            TraktApplication.fetchTrendingShows(showsObj, new TrendingShowCallback());
```

Comment:
nit: you can save these callbacks into instance variables that way you only ever create them once